installer/install.iss

```diff
@@ -82,7 +82,7 @@ InfoBeforeFile={#SourcePath}\..\gpl-2.0.rtf
 #ifdef OUTPUT_TO_TEMP
 PrivilegesRequired=lowest
 #else
-PrivilegesRequired=none
+PrivilegesRequired=admin
```

Comment:
The commit message gives enough context to understand what this does, but I am missing an explanation as to _why_ we want to do that. Like, why not leave it with the undocumented behavior?

Looking through the commit history, I see that 1f3e1f7e360ca997f877cece0c0eb5cb1927a198 followed the advice to replace `none` with `lowest`. And 23672af723da18e5bc3c679e52106de3c2dec55a reverted that choice. In particular the latter's commit message makes me dubious whether we considered all the consequences of changing to `admin`.

Reply:
Also: do we want https://jrsoftware.org/ishelp/topic_setup_usepreviousprivileges.htm? And what about https://jrsoftware.org/ishelp/topic_setup_privilegesrequiredoverridesallowed.htm?

Again, I am not opposed to merge this PR, I just need to make sure that the change is sound, i.e. well-researched. We cannot unleash a broken installer onto potentially millions of users.